docs/getting-started.md

```diff
@@ -216,75 +216,58 @@ Troubleshooting tips if your EV3 won't boot
-* __USB Wi-Fi Dongle__
-    * Setting up Wi-Fi Via the Brickman User Interface (hopefully you can figure
-      this out, but it would be nice if someone made a tutorial)
-    * [Setting Up Wi-Fi Using the Command Line](../tutorials/setting-up-wifi-using-the-command-line)
-      (requires another connection type first to get to the command line, but it's way
-      easier to enter your passphrase this way)
-* __Bluetooth__
-    * Note: Bluetooth may not work on the first boot. Please reboot if you see "???"
-      after you power on Bluetooth.
-    * [Connecting to the Internet](../tutorials/connecting-to-the-internet-via-bluetooth) tutorial
+There are lots of choices here. Choose the connection option that best suits you
+and your available devices.
 
 {% include icon.html type="info" %}
 For Raspberry Pi with no display, you must use the wired Ethernet port
 to connect for the first time. You can set up additional connections using the
 `connmanctl` command.
 {: class="alert alert-info"}
 
-Now that you have a network connection, you should
-[connect to your EV3 with SSH](/docs/tutorials/connecting-to-ev3dev-with-ssh)
-if you haven't done so already.
+<a class="btn btn-md btn-primary" href="/docs/networking"><span class="glyphicon glyphicon-globe"></span> Check out the available networking options</a>
 
 </div>
 </div>
 
 <div class="colored-section">
 <div class="container" markdown="1">
 
-## Step 6: Do something awesome
+## Step 6: Connect to the EV3 via SSH
 
-The brick can run almost all programming languages that any other Linux distro can, so your
-favorite language is probably supported. Language bindings have already been
-written for many languages. **You can learn more about the available libraries
-[here](/docs/programming-languages).**
+Now that you have a network connection, you should connect to your EV3 with SSH.
+Connecting allows you to verify that the networking option that you configured
+in the previous step is working. Once you have an SSH connection, you will be
+able to securely run terminal commands on the EV3 from your PC. This lets you
+run programs, change settings, and install new programs.
 
-If the language you want isn't listed, you still can use it, but you'll have to
-do more of the heavy lifting yourself. You can look at the [driver index page](/docs/drivers/)
-for information on the interfaces you need to use to control devices. Once you get the
-hang of it, you can even write your own interface library and have it listed here!
+<a class="btn btn-md btn-primary" href="/docs/tutorials/connecting-to-ev3dev-with-ssh"><span class="glyphicon glyphicon-transfer"></span> Learn how to connect with SSH</a>
+
+To test this out, let's try running a command. Type (or copy-and-paste) the
+following into the terminal, and press enter. It will use the speakers and
+text-to-speech software to speak out loud. **If you are not on an EV3, this
+will not work. It requires the EV3's built-in speaker.**
```

Comment:
> This guide currently doesn't support the EVB

Sure it does, even if it is not explicitly mentioned.


> In general, I think that whatever demo we put here should operate without any external hardware

Perhaps we should install the `fortune` package by default?